packages/storybook/stories/react-inbox/with-toast.stories.tsx

```diff
@@ -52,6 +52,7 @@ export function Default() {
         clientKey={clientKey}
         userId={userId}
         transport={courierTransport}
+        brandId="S0E8PFMXC44SSFG6J8A7EF4QXRNX"
```

Comment:
might want an env variable so we can change between our dev envs

Reply:
Done